app/src/main/java/de/westnordost/streetcomplete/quests/diet_type/AddVegan.java

```diff
@@ -0,0 +1,43 @@
+package de.westnordost.streetcomplete.quests.diet_type;
+
+import android.os.Bundle;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+
+public class AddVegan extends SimpleOverpassQuestType
+{
+	@Inject public AddVegan(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override protected String getTagFilters()
+	{
+		return "nodes, ways with ( amenity ~ restaurant|cafe|pub|fast_food|bar ) and diet:vegetarian and !diet:vegan";
```

Comment:
`diet:vegetarian~yes|only`
Because this quest shouldn't show if the user selected "no" in the vegetarian quest

Also, superfluous parenthesis.